src/main/java/Duke.java

```diff
@@ -1,10 +1,57 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String logo = "____________________________________________________________\n"
```

Comment:
```suggestion
        String welcomeMessage = "____________________________________________________________\n"
```
Perhaps rename logo to welcomeMessage so that it would better describe the variable?